### DISCUSSION TITLE
Curso de Node - MLab não está disponivel para acesso

### DISCUSSION THREAD

**Discussion by MalinaliRap** [balta-io/forum#355] [Tira Dúvidas] [answered]
Estou fazendo o curso de Node na plataforma https://balta.io/, estou na aula intitulada "MongoDb Setup", porém a ferramenta MLab mostrada nesta aula não está mais disponivel, queria saber se existe uma ferramente alternativa para utilizar o mongodb em nuvem, estou tentando utilizar o Atlas, porém a ferramente é muito diferente da mostrada em aula. Desde já agradeço a atenção.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @MalinaliRap, você pode rodar o Mongo localmente com Docker!
https://balta.io/blog/mongodb-docker
